Resources/Prototypes/Floof/body/Organs/Friendstomach.yml

```diff
@@ -0,0 +1,25 @@
+﻿- type: entity
+  id: OrganFriendStomach
+  parent: OrganAnimalStomach
+  noSpawn: true
+  components:
+  - type: Stomach
+    specialDigestible:
+      tags:
+      - IceCream #Floofstation
+      - Fruit
+      - ReptilianFood
+      - Meat
+      - Pill
+      - Crayon
+      - Paper
+      - ClothMade
+  - type: SolutionContainerManager
+    solutions:
+      stomach:
+        maxVol: 500
```

Comment:
sure I can do that, it was moreso exagurated to see if it would work to fix the issue, On my test server it seems it did. so I'm fine making it like 100

Reply:
Has this been changed to ~100?